pkg/quota/registry.go

```diff
@@ -7,14 +7,16 @@ import (
 	"github.com/openshift/origin/pkg/quota/image"
 )
 
-// NewRegistry returns a registry object that knows how to evaluate quota usage of OpenShift resources.
-// Registry for resource quota controller cannot be used with resource quota admission plugin and
-// vice versa. If the registry will be used in admission plugin, pass true to forAdmission.
-// See a package documentation of pkg/quota/image for more details.
-func NewRegistry(osClient osclient.Interface, forAdmission bool) kquota.Registry {
-	if forAdmission {
-		return image.NewImageRegistryForAdmission(osClient)
-	} else {
-		return image.NewImageRegistry(osClient)
-	}
+// NewOriginQuotaRegistry returns a registry object that knows how to evaluate quota usage of OpenShift
```

Comment:
I've been assisting with other blockers. I'm hoping to get some review time
this weekend but it will likely be minimal. And I'm on PTO Monday.

On Fri, Apr 1, 2016 at 3:44 PM, Clayton Coleman <notifications@github.com>
wrote:

> In pkg/quota/registry.go
> <https://github.com/openshift/origin/pull/8195#discussion_r58257427>:
>
> > @@ -7,14 +7,16 @@ import (
> >  	"github.com/openshift/origin/pkg/quota/image"
> >  )
> >
> > -// NewRegistry returns a registry object that knows how to evaluate quota usage of OpenShift resources.
> > -// Registry for resource quota controller cannot be used with resource quota admission plugin and
> > -// vice versa. If the registry will be used in admission plugin, pass true to forAdmission.
> > -// See a package documentation of pkg/quota/image for more details.
> > -func NewRegistry(osClient osclient.Interface, forAdmission bool) kquota.Registry {
> > -	if forAdmission {
> > -		return image.NewImageRegistryForAdmission(osClient)
> > -	} else {
> > -		return image.NewImageRegistry(osClient)
> > -	}
> > +// NewOriginQuotaRegistry returns a registry object that knows how to evaluate quota usage of OpenShift
>
> Will try to come back to this before monday.
> … <#m_-1040364610014226320_>
> On Fri, Apr 1, 2016 at 2:47 PM, Andy Goldstein <notifications@github.com>
> wrote: In pkg/quota/registry.go <#8195 (comment)
> <https://github.com/openshift/origin/pull/8195#discussion_r58250057>>: >
> @@ -7,14 +7,16 @@ import ( > "github.com/openshift/origin/pkg/quota/image"
> > ) > > -// NewRegistry returns a registry object that knows how to
> evaluate quota usage of OpenShift resources. > -// Registry for resource
> quota controller cannot be used with resource quota admission plugin and >
> -// vice versa. If the registry will be used in admission plugin, pass true
> to forAdmission. > -// See a package documentation of pkg/quota/image for
> more details. > -func NewRegistry(osClient osclient.Interface, forAdmission
> bool) kquota.Registry { > - if forAdmission { > - return
> image.NewImageRegistryForAdmission(osClient) > - } else { > - return
> image.NewImageRegistry(osClient) > - } > +// NewOriginQuotaRegistry returns
> a registry object that knows how to evaluate quota usage of OpenShift Or do
> we want to keep it so we can add other things to the registry as we quota
> more than just image related bits? — You are receiving this because you
> were mentioned. Reply to this email directly or view it on GitHub <
> https://github.com/openshift/origin/pull/8195/files/5f0bfc04c89b90227820aefc80d3d42dcd86b9dd#r58250057>
>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8195/files/5f0bfc04c89b90227820aefc80d3d42dcd86b9dd#r58257427>
>